akka-remote/src/main/resources/reference.conf

```diff
@@ -249,6 +249,10 @@ akka {
             advertisement-interval = 1 minute # TODO find good number as default, for benchmarks trigger immediately
           }
         }
+
+        # Remote instruments which should be initialized and used for remote messages.
+        # Refer to `akka.remote.artery.RemoteInstrument` for more information.
+        instruments = ${?akka.remote.artery.advanced.instruments} []
```

Comment:
This allows for "auto loading extensions". If someone wanted to they could ship with their reference.conf which has their instruments defined in it and it would resolve properly. We do this with `  library-extensions = ${?akka.library-extensions} []` too